<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>471</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>439</width>
    <height>471</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>439</width>
    <height>471</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление данных в базу</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>421</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Пароль:</string>
   </property>
   <widget class="QLineEdit" name="pwd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="repwd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="view_pwd">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль еще раз:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="prBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>190</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>421</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Адрес и логин</string>
   </property>
   <widget class="QLineEdit" name="inet_addr">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="login">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Интернет аддрес:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить данные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ext">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Группа:</string>
   </property>
   <widget class="QComboBox" name="cbb1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>331</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_group">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+Группу</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ext</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
